tic_tac_toe_board.py

```diff
@@ -0,0 +1,47 @@
+class Board():
+    def __init__(self):
+        self.spaces = [None] * 9
+
+    def is_empty_at(self, space):
+        return self.spaces[space] is None
+
+    def mark(self, space, symbol):
+        if self.is_empty_at(space):
+            self.spaces[space] = symbol
+        else:
+            raise ValueError("Invalid Move !")
+
+    def __getitem__(self,space):
+         return self.spaces[space]
+
+    def is_a_way_to_win_a_board(self):
```

Comment:
Is this really what you want to be able to ask the board? It looks like this answers "who won".